### PR TITLE
Fat JAR with Shadow clarifications

### PR DESCRIPTION
**Pull request by clarksandholtz**
Added a link to Shadow's docs.

Added what you need to add to `build.gradle` in order to actually generate the JAR.

### CODE REVIEW

**Comment by clarksandholtz** [comment]
I changed the class from `com.your.MainClass` to `io.ktor.samples.hello.HelloApplicationKt` and kept the comment as seen in the [docs for Proguard](https://ktor.io/servers/deploy/proguard.html) to increase consistency.

File: servers/deploy/packing/0fatjar.md [hunk 1]
```diff
@@ -37,6 +37,13 @@ apply plugin: 'application'
 
 //mainClassName = 'io.ktor.server.netty.DevelopmentEngine' // For versions < 1.0.0-beta-3
 mainClassName = 'io.ktor.server.netty.EngineMain' // Starting with 1.0.0-beta-3
+
+// This task will generate your fat JAR and put it in the ./build/libs/ directory
+shadowJar {
+    manifest {
+        attributes 'Main-Class': 'io.ktor.samples.hello.HelloApplication'
```
Comment (cy6erGn0m) [inline servers/deploy/packing/0fatjar.md]: In this particular example we are using `EngineMain`, not `HelloApplication`

**Comment by cy6erGn0m** [comment]
Thanks!

**Comment by clarksandholtz** [comment]
You're welcome! I hope to be able to provide more meaningful contributions soon!

**Pull request merged**